types/express-mung/index.d.ts

```diff
@@ -8,37 +8,50 @@
 /// <reference types="node"/>
```

Comment:
☝️
> // Type definitions for express-mung 0.4.2

Update version comment to 0.5.1?

Reply:
👍 thanks good catch! I've pushed an update correcting the comment.